### ISSUE TITLE
Recording causing immense web UI sluggish behavior into timeout

### ISSUE THREAD

**Issue by akumar95** [MrBrax/LiveStreamDVR#327]
Hello,

So I have this problem that when a stream is recorded, instantly the web UI hangs and ultimately leading to error 523 timeouts. The docker compose file I'm running is:

```
version: "3.8"
services:
  web:
    build: .
    image: mrbrax/twitchautomator:master
    ports:
      - "8082:8080"
    volumes:
      - /volume1/Data/twitchauto:/usr/local/share/twitchautomator/data
    environment:
      - NODE_ENV=production
```
The VPS LivestreamDVR has said specs:
```
AMD EPYC 7J13 CPU
96GB RAM
Ubuntu 22.04
```

The hard drive space on this VPS isn't as much as I'd like so I have my Synology mounted to the VPS so I have 14TB of space to use. I've tried mounting it via NFS and SMB but the webpage still hangs unless I restart the container. Looking at the logs the application has no issues continuing the recording but the page is where the problem persists. 

Would the issue be caused by my Synology or somewhere in my network?

**Comment by MrBrax** [comment]
is the webpage creating a large amount of logging or network requests?

**Comment by MrBrax** [comment]
also not sure why a timeout is 523, 500+ is for serverside

**Comment by akumar95** [comment]
I’m actually unsure if the web page is creating large amounts of logging or network requests. Logging is set to default and no changes to networking I believe?

Also my apologies, meant host timeout is the error. Running cloudflare DNS

**Comment by MrBrax** [comment]
so the server becomes unresponsive and not the web interface? does it continue logging and all that?

**Comment by akumar95** [comment]
Oh yes, apologies. The server becomes unresponsive. The application has no problem continuing to log and record the stream, however. The server just stalls unless restarted. 

Regarding channels, there are only 4 that I have added.

**Comment by MrBrax** [comment]
probably related to #275 then which i haven't encountered since.

any kind of logging when it happens?

**Comment by MrBrax** [comment]
i might have an idea, could be unresolved http requests that fill up the queue. now when you enable debug mode and restart it should print current connections every 30 seconds

**Comment by akumar95** [comment]
Sorry for the late response. Here is the log from LivestreamDVR picking up the streamer is live and the server becoming unresponsive:

```
2022-10-08 21:23:20.915 28 | helper <ERROR> Could not create thumbnail for user logo for 39656400: undefined
2022-10-08 21:23:22.562 28 | channel <DEBUG> Fetching user data for login stickasaurusrex, force: false
2022-10-08 21:23:22.779 28 | channel <DEBUG> User data found in memory cache for login stickasaurusrex
2022-10-08 21:23:23.006 28 | channel <DEBUG> Returning memory cache for login stickasaurusrex
2022-10-08 21:23:23.216 28 | channel <DEBUG> Fetching user data for login stickasaurusrex, force: false
2022-10-08 21:23:23.437 28 | channel <DEBUG> User data found in memory cache for login stickasaurusrex
2022-10-08 21:23:23.657 28 | channel <DEBUG> Returning memory cache for login stickasaurusrex
2022-10-08 21:23:23.873 28 | dvr <INFO> Saving channel config
2022-10-08 21:23:24.343 28 | channel <DEBUG> Load from login stickasaurusrex
2022-10-08 21:23:24.560 28 | channel <DEBUG> Fetching user data for login stickasaurusrex, force: false
2022-10-08 21:23:24.780 28 | channel <DEBUG> User data found in memory cache for login stickasaurusrex
2022-10-08 21:23:24.999 28 | channel <DEBUG> Returning memory cache for login stickasaurusrex
2022-10-08 21:23:25.216 28 | channel <DEBUG> Load channel 39656400
2022-10-08 21:23:25.443 28 | channel <DEBUG> Fetching user data for id 39656400, force: false
2022-10-08 21:23:25.672 28 | channel <DEBUG> User data found in memory cache for id 39656400
2022-10-08 21:23:25.886 28 | channel <DEBUG> Returning memory cache for id 39656400
2022-10-08 21:23:26.206 28 | vodclass <DEBUG> Found 0 clips for stickasaurusrex
2022-10-08 21:23:26.425 28 | channel <DEBUG> Fetching channel data for 39656400
2022-10-08 21:23:26.683 28 | helper <DEBUG> Fetched access token from cache
2022-10-08 21:23:27.146 28 | helper <WARNING> Game id 509673 not in cache.
2022-10-08 21:23:27.543 28 | channel <INFO> Updated chapter data for stickasaurusrex
2022-10-08 21:23:27.769 28 | channel <DEBUG> Fetching user data for id 39656400, force: false
2022-10-08 21:23:27.980 28 | channel <DEBUG> User data found in memory cache for id 39656400
2022-10-08 21:23:28.195 28 | channel <DEBUG> Returning memory cache for id 39656400
2022-10-08 21:23:28.415 28 | helper <INFO> Subscribe to 39656400:stream.online (stickasaurusrex)
2022-10-08 21:23:29.521 28 | helper <SUCCESS> Subscribe for 39656400:stream.online (stickasaurusrex) sent. Check logs for a 'subscription active' message.
2022-10-08 21:23:29.723 28 | helper <INFO> Subscribe to 39656400:stream.offline (stickasaurusrex)
2022-10-08 21:23:29.979 28 | hook <INFO> Hook called with message ID b1213b98-765d-4332-ab45-141343be4aa4, version 1, type stream.online (retry 0, type webhook_callback_verification, date 2022-10-08T21:23:28.731116847Z)
2022-10-08 21:23:30.203 28 | channel <DEBUG> Fetching user data for id 39656400, force: false
2022-10-08 21:23:30.425 28 | channel <DEBUG> User data found in memory cache for id 39656400
2022-10-08 21:23:30.636 28 | channel <DEBUG> Returning memory cache for id 39656400
2022-10-08 21:23:30.847 28 | hook <INFO> Challenge received for 39656400:stream.online (stickasaurusrex) (64eeb222-af66-4f74-a138-f851171934f5), retry 0
2022-10-08 21:23:31.058 28 | hook <SUCCESS> Challenge completed, subscription active for 39656400:stream.online (stickasaurusrex) (64eeb222-af66-4f74-a138-f851171934f5), retry 0.
2022-10-08 21:23:32.127 28 | helper <SUCCESS> Subscribe for 39656400:stream.offline (stickasaurusrex) sent. Check logs for a 'subscription active' message.
2022-10-08 21:23:32.342 28 | helper <INFO> Subscribe to 39656400:channel.update (stickasaurusrex)
2022-10-08 21:23:32.658 28 | hook <INFO> Hook called with message ID e68df6bb-2e14-45a0-ba4a-02b23e474bdb, version 1, type stream.offline (retry 0, type webhook_callback_verification, date 2022-10-08T21:23:30.044141526Z)
2022-10-08 21:23:32.862 28 | channel <DEBUG> Fetching user data for id 39656400, force: false
2022-10-08 21:23:33.071 28 | channel <DEBUG> User data found in memory cache for id 39656400
2022-10-08 21:23:33.276 28 | channel <DEBUG> Returning memory cache for id 39656400
2022-10-08 21:23:33.500 28 | hook <INFO> Challenge received for 39656400:stream.offline (stickasaurusrex) (d2a68ccb-2d9e-489e-bcfe-7fbe28015869), retry 0
2022-10-08 21:23:33.725 28 | hook <SUCCESS> Challenge completed, subscription active for 39656400:stream.offline (stickasaurusrex) (d2a68ccb-2d9e-489e-bcfe-7fbe28015869), retry 0.
2022-10-08 21:23:34.969 28 | helper <SUCCESS> Subscribe for 39656400:channel.update (stickasaurusrex) sent. Check logs for a 'subscription active' message.
2022-10-08 21:23:35.290 28 | helper <INFO> Querying streams for streamer id 39656400
2022-10-08 21:23:35.506 28 | route.channels.add <SUCCESS> Created channel: stickasaurusrex
2022-10-08 21:23:35.727 28 | hook <INFO> Hook called with message ID 90d0eac2-91ea-45f4-9dc5-0bd9d4520467, version 1, type channel.update (retry 0, type webhook_callback_verification, date 2022-10-08T21:23:34.181504704Z)
2022-10-08 21:23:35.937 28 | channel <DEBUG> Fetching user data for id 39656400, force: false
2022-10-08 21:23:36.153 28 | channel <DEBUG> User data found in memory cache for id 39656400
2022-10-08 21:23:36.373 28 | channel <DEBUG> Returning memory cache for id 39656400
2022-10-08 21:23:36.591 28 | hook <INFO> Challenge received for 39656400:channel.update (stickasaurusrex) (20d445d0-99a0-4322-ae9b-fbac5b25e286), retry 0
2022-10-08 21:23:36.808 28 | hook <SUCCESS> Challenge completed, subscription active for 39656400:channel.update (stickasaurusrex) (20d445d0-99a0-4322-ae9b-fbac5b25e286), retry 0.
2022-10-08 21:23:37.385 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:23:39.328 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:23:39.548 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:23:39.760 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:08.914 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:11.262 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:11.495 28 | hook <INFO> Hook called with message ID eQa0djxYZjkrMlVVKGf3g7r6gSm28JwIfrFETdBVLRA=, version 1, type channel.update (retry 0, type notification, date 2022-10-08T21:24:09.081489334Z)
2022-10-08 21:24:11.716 28 | hook <INFO> Dumping debug hook payload to /usr/local/share/twitchautomator/data/payloads/tw_2022_10_08T21_24_11_716Z_channel.update.json
2022-10-08 21:24:12.102 28 | hook <DEBUG> Signature checked, no challenge, retry 0. Run handle...
2022-10-08 21:24:12.329 28 | automator.handle <DEBUG> Handle called, proceed to parsing.
2022-10-08 21:24:12.976 28 | automator.handle <INFO> Automator channel.update event for stickasaurusrex
2022-10-08 21:24:13.202 28 | automator <INFO> Channel stickasaurusrex not capturing, saving channel data to cache: Makers & Crafting (Getting Some Helmets Ready To Clear! Come Hang Out!)
2022-10-08 21:24:13.748 28 | job <DEBUG> Check status for job capture_blainethepaintv_40321136425
2022-10-08 21:24:14.058 28 | notify <DEBUG> Pushover response
2022-10-08 21:24:21.950 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:22.566 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:34.617 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:34.864 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:40.119 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:40.852 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:24:41.180 28 | job <DEBUG> Check status for job capture_blainethepaintv_40321136425
2022-10-08 21:24:43.724 28 | helper <INFO> Querying streams for streamer id 39656400
2022-10-08 21:24:43.965 28 | automator <INFO> Channel stickasaurusrex is still offline, timeout check.
2022-10-08 21:24:48.146 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:25:05.886 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:25:20.876 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:25:21.289 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:25:21.510 28 | vodclass <DEBUG> Can't request duration because blainethepaintv_2022-10-08T21_21_19Z_40321136425 is still recording!
2022-10-08 21:25:54.068 28 | hook <INFO> Hook called with message ID P9QDI1Q2RbS_GNVFVK1cxAXExpbiSBL8Qt9tPxUguTM=, version 1, type stream.online (retry 0, type notification, date 2022-10-08T21:25:53.780544385Z)
2022-10-08 21:25:54.286 28 | hook <INFO> Dumping debug hook payload to /usr/local/share/twitchautomator/data/payloads/tw_2022_10_08T21_25_54_286Z_stream.online.json
2022-10-08 21:25:54.626 28 | hook <DEBUG> Signature checked, no challenge, retry 0. Run handle...
2022-10-08 21:25:54.849 28 | automator.handle <DEBUG> Handle called, proceed to parsing.
2022-10-08 21:25:55.571 28 | automator.handle <INFO> Automator stream.online event for stickasaurusrex (retry 0)
2022-10-08 21:25:56.213 28 | automator.handle <INFO> stickasaurusrex stream has ID 40321153401, started 2022-10-08T21:24:44Z
2022-10-08 21:25:57.113 28 | automator <DEBUG> Making folder for stickasaurusrex_2022-10-08T21_24_44Z_40321153401.
2022-10-08 21:25:57.369 28 | channel <INFO> Create VOD JSON for stickasaurusrex: stickasaurusrex_2022-10-08T21_24_44Z_40321153401.json @ /usr/local/share/twitchautomator/data/storage/vods/stickasaurusrex/stickasaurusrex_2022-10-08T21_24_44Z_40321153401
2022-10-08 21:25:57.575 28 | vodclass <SUCCESS> Saving JSON of stickasaurusrex_2022-10-08T21_24_44Z_40321153401  (create json)
2022-10-08 21:25:58.514 28 | channel <DEBUG> Fetching user data for id 39656400, force: false
2022-10-08 21:25:58.723 28 | channel <DEBUG> User data found in memory cache for id 39656400
2022-10-08 21:25:58.933 28 | channel <DEBUG> Returning memory cache for id 39656400
2022-10-08 21:25:59.148 28 | channel <DEBUG> Fetching user data for id 39656400, force: false
2022-10-08 21:25:59.361 28 | channel <DEBUG> User data found in memory cache for id 39656400
2022-10-08 21:25:59.581 28 | channel <DEBUG> Returning memory cache for id 39656400
2022-10-08 21:25:59.800 28 | vodclass <ERROR> No chapters on stickasaurusrex_2022-10-08T21_24_44Z_40321153401!
2022-10-08 21:26:00.010 28 | vodclass <WARNING> Loaded VOD stickasaurusrex_2022-10-08T21_24_44Z_40321153401 is not finalized!
2022-10-08 21:26:00.588 28 | vodclass <DEBUG> Can't request duration because stickasaurusrex_2022-10-08T21_24_44Z_40321153401 is converting!
2022-10-08 21:26:00.815 28 | vodclass <WARNING> Saving JSON of stickasaurusrex_2022-10-08T21_24_44Z_40321153401 with no chapters!!
2022-10-08 21:26:01.040 28 | vodclass <SUCCESS> Saving JSON of stickasaurusrex_2022-10-08T21_24_44Z_40321153401  (is_capturing set)
2022-10-08 21:26:01.425 28 | automator <INFO> Update game for stickasaurusrex_2022-10-08T21_24_44Z_40321153401
2022-10-08 21:26:01.635 28 | automator <SUCCESS> Channel data for stickasaurusrex fetched from cache.
2022-10-08 21:26:01.851 28 | helper <WARNING> Game id 509673 not in cache.
2022-10-08 21:26:02.074 28 | helper <DEBUG> Game id 509673 not in cache, fetching...
2022-10-08 21:26:02.287 28 | vodclass <WARNING> Saving JSON of stickasaurusrex_2022-10-08T21_24_44Z_40321153401 with no chapters!!
2022-10-08 21:26:02.498 28 | vodclass <SUCCESS> Saving JSON of stickasaurusrex_2022-10-08T21_24_44Z_40321153401  (dt_capture_started set)
2022-10-08 21:26:02.708 28 | automator <INFO> Starting capture with filename stickasaurusrex_2022-10-08T21_24_44Z_40321153401.ts
2022-10-08 21:26:02.929 28 | automator <SUCCESS> Spawned process 104 for capture_stickasaurusrex_40321153401
2022-10-08 21:26:03.215 28 | job <DEBUG> Set PID 104 for job capture_stickasaurusrex_40321153401
2022-10-08 21:26:03.446 28 | job <DEBUG> Set process for job capture_stickasaurusrex_40321153401
2022-10-08 21:26:04.126 28 | job <DEBUG> Start log for job capture_stickasaurusrex_40321153401 on path /usr/local/share/twitchautomator/data/logs/software/capture_stickasaurusrex_40321153401
2022-10-08 21:26:05.744 28 | job <DEBUG> Attach log for job capture_stickasaurusrex_40321153401 to process
2022-10-08 21:26:05.960 28 | job <INFO> Save job capture_stickasaurusrex_40321153401 with PID 104 to /usr/local/share/twitchautomator/data/cache/pids/capture_stickasaurusrex_40321153401.json
2022-10-08 21:26:06.285 28 | job <DEBUG> Job capture_stickasaurusrex_40321153401 saved
2022-10-08 21:26:06.487 28 | job <DEBUG> New job capture_stickasaurusrex_40321153401 added to jobs list
2022-10-08 21:26:06.703 28 | vodclass <DEBUG> Can't request duration because stickasaurusrex_2022-10-08T21_24_44Z_40321153401 is still recording!
2022-10-08 21:26:25.150 28 | job <DEBUG> Spawned process for job capture_stickasaurusrex_40321153401
2022-10-08 21:26:25.414 28 | vodclass <DEBUG> Can't request duration because stickasaurusrex_2022-10-08T21_24_44Z_40321153401 is converting!
2022-10-08 21:27:31.019 28 | automator <INFO> Stream resolution for stickasaurusrex_2022-10-08T21_24_44Z_40321153401: 720p
2022-10-08 21:27:31.269 28 | automator <WARNING> Stream resolution 720p assumed to not be in channel quality list
2022-10-08 21:27:31.485 28 | automator <INFO> Capturing of stickasaurusrex_2022-10-08T21_24_44Z_40321153401, will try to remove ads!
2022-10-08 21:27:31.705 28 | automator <INFO> Streamlink now writing output to container.
2022-10-08 21:27:32.323 28 | vodclass <DEBUG> Can't request duration because stickasaurusrex_2022-10-08T21_24_44Z_40321153401 is still recording!
WebSocket payload job_update dispatching...
No clients connected to broker for data: {"server":true,"action":"job_update","data":{"job_name":"capture...
```
At this point, the server becomes unresponsive and refreshing leads to an error 523. I would have to force restart the container to bring the application back.

**Comment by MrBrax** [comment]
forgot to say, you gotta change to develop-ts branch for that update to work

doesn't seem to be anything visible from the server itself so...

**Comment by akumar95** [comment]
Hmm yea was on the develop branch. Restarted and an pushover API timeout occurred:

```
2022-10-08 22:01:34.845 | vodclass <DEBUG> VOD JSON chiqszn_2022-10-08T22_00_30Z_47273506141 exists (again?) change
VOD file /usr/local/share/twitchautomator/data/storage/vods/chiqszn/chiqszn_2022-10-08T22_00_30Z_47273506141/chiqszn_2022-10-08T22_00_30Z_47273506141.json changed (external/change)!
2022-10-08 22:01:35.023 | vodclass <DEBUG> VOD JSON chiqszn_2022-10-08T22_00_30Z_47273506141 exists (again?) change
2022-10-08 22:01:35.239 | vodclass <DEBUG> Can't request duration because chiqszn_2022-10-08T22_00_30Z_47273506141 is still recording!
WebSocket payload vod_updated dispatching...
No clients connected to broker for data: {"server":true,"action":"vod_updated","data":{"vod":{"provider":...
2022-10-08 22:01:40.548 | notify <ERROR> Pushover axios error request: Client network socket disconnected before secure TLS connection was established ([object Object])
😤 Error stringifying log data! {
  module: 'notify',
  time: 1665266500654,
  level: 'ERROR',
  text: 'Pushover axios error request: Client network socket disconnected before secure TLS connection was established ([object Object])',
  pid: 28,
  g: '223ead22932d4305b1246cfbc0beb9533b2a8d2c',
  metadata: {
    err: Error: Client network socket disconnected before secure TLS connection was established
        at connResetException (node:internal/errors:711:14)
        at TLSSocket.onConnectEnd (node:_tls_wrap:1593:19)
        at TLSSocket.emit (node:events:525:35)
        at endReadableNT (node:internal/streams/readable:1359:12)
        at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
      code: 'ECONNRESET',
      path: null,
      host: 'api.pushover.net',
      port: 443,
      localAddress: undefined,
      config: [Object],
      request: [Writable],
      response: undefined,
      isAxiosError: true,
      toJSON: [Function: toJSON]
    },
    request: Writable {
      _writableState: [WritableState],
      _events: [Object: null prototype],
      _eventsCount: 3,
      _maxListeners: undefined,
      _options: [Object],
      _ended: false,
      _ending: true,
      _redirectCount: 0,
      _redirects: [],
      _requestBodyLength: 199,
      _requestBodyBuffers: [Array],
      _onNativeResponse: [Function (anonymous)],
      _currentRequest: [ClientRequest],
      _currentUrl: 'https://api.pushover.net/1/messages.json',
      [Symbol(kCapture)]: false
    }
  }
}
Pushover axios error request: Client network socket disconnected before secure TLS connection was established ([object Object]) {
  "message": "Client network socket disconnected before secure TLS connection was established",
  "name": "Error",
  "stack": "Error: Client network socket disconnected before secure TLS connection was established\n    at connResetException (node:internal/errors:711:14)\n    at TLSSocket.onConnectEnd (node:_tls_wrap:1593:19)\n    at TLSSocket.emit (node:events:525:35)\n    at endReadableNT (node:internal/streams/readable:1359:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)",
  "config": {
    "transitional": {
      "silentJSONParsing": true,
      "forcedJSONParsing": true,
      "clarifyTimeoutError": false
    },
    "transformRequest": [
      null
    ],
    "transformResponse": [
      null
    ],
    "timeout": 0,
    "xsrfCookieName": "XSRF-TOKEN",
    "xsrfHeaderName": "X-XSRF-TOKEN",
    "maxContentLength": -1,
    "maxBodyLength": -1,
    "headers": {
      "Accept": "application/json, text/plain, */*",
      "Content-Type": "application/json",
      "User-Agent": "axios/0.26.0",
      "Content-Length": 199
    },
    "method": "post",
    "url": "https://api.pushover.net/1/messages.json",
    "data": "{\"token\":\"[SNIP]\",\"user\":\"[SNIP]",\"title\":\"chiqszn is now playing Fortnite!\",\"message\":\"Squad ZB Cup | New !montage\",\"url\":\"https://twitch.tv/chiqszn\"}"
  },
  "code": "ECONNRESET",
  "status": null
}
```

**Comment by MrBrax** [comment]
![1665299411964](https://user-images.githubusercontent.com/1517911/194743043-348cd661-a487-422d-b8e0-b0fa8897cfb6.png)

this should be output if debug mode is enabled

**Comment by akumar95** [comment]
Hey small update. I purchased a VPS with a large storage so see if it was a networking issue. Ran it for a day recording several streams and no server issues. 

Turned out it was a really slow connection with my VPS and the synology causing the server issues. I have a VPS located in a different state so I use Tailscale so connect to two. After upgrading to DSM7 tailscale had no outgoing connection but after following said steps: https://tailscale.com/kb/1131/synology/#enabling-synology-outbound-connections

The connection now runs at full speed on the server LiveStreamDVR runs and my synology. All is good :)
